requirements_docs.txt

```diff
@@ -15,7 +15,6 @@ sphinx-copybutton
 sphinx-gallery>=0.8.1
 lxml
 trimesh
-vtk==8.1.2  # needed because 9.0.X has an issue with the plotter closing with iren
```

Comment:
I think we still cannot solve the issue of #185 . The error did not occured?

Reply:
Error still occurs, but the doc build is now being built on the CI twice with:
https://github.com/pyvista/pyvista/blob/2205db90ef072b88172d208a8dafd84fd5972157/azure-pipelines.yml#L248

We need VTK9 for the new physically based rendering examples.  I'm fine with adding it back in if people need to build docs locally on their machines, but they'll instead get failures when attempting to build `examples/02-plot/pbr.py`.